charts/deployment-operator/values.yaml

```diff
@@ -5,6 +5,32 @@
 additionalVolumes: ~
 additionalVolumeMounts: ~
 
+# Allows mounting custom certs to deployment operator.
+# Certificates are mounted to the "/etc/ssl/certs" directory.
+# An array of files can be provided. Every file will be mounted
+# on a specific subpath to ensure that multiple files can be mounted
+# without overriding each other.
+#
+# In order to override default "ca-certificates.crt" simply
+# provide a "file: ca-certificates.crt" when providing custom certs.
+# It will be mounted on a subpath to override only this single file.
+#
+# Example usage:
+#  - name: ca-pemstore
+#    create: true
+#    certData: xyz
+#    file: my-cert.pem
+#    volume:
```

Comment:
could we just infer this entirely so users don't have to back infer the naming conventions for the generated configmaps?  (I don't think the persistentVolumeClaim case is that plausible since the purpose of this is to have the deploy operator be able to work on fresh boot with a PKI setup)

Reply:
How would you want the values to look like? Just support cert injection via config map and that's it?

Should it simply look like this?
```
certs:
  - name: ca-pemstore
    certData: xyz
    file: my-cert.pem
```

Reply:
yea that looks fine to me, and make sure the configmap and file wirings are all clean from there.

Reply:
done